apps/web/src/lang/fr.json

```diff
@@ -12062,6 +12073,10 @@
     "defaultMessage": "Processus hors plateforme",
     "description": "Off-platform section header"
   },
+  "trerKD": {
```

Comment:
Because the French string is identical to its English equivalent, this should be removed from this file and `- trerKD # Type` be added to the whitelist file: `apps/web/src/lang/whitelist.yml`.